src/routes/demos/.form/.BackgroundPoster.svelte

```diff
@@ -0,0 +1,225 @@
+        class="icon is-hidden-mobile  informationIcon is-small "
+      >
+        <i class="mdi mdi-18px mdi-information-outline " />
+        <p class="is-size-7 pl-1 ">
+          {displayFileSizeError ? 'File size is too large' : `File must not exceed ${maxFileSize / 1000}KB in size`}
+        </p>
+      </div>
+    </div>
+  {/if}
+
+  <section class="formSlot">
+    <slot />
+  </section>
+</div>
+
+<style lang="scss">
+  .informationIcon {
+    width: 100%;
+    justify-content: flex-start;
+    z-index: 1;
+  }
+
+  .formSlot {
+    display: flex;
+    position: absolute;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-between;
+    width: 100%;
+    height: 100%;
+  }
+
+  .content {
+    display: flex;
+    position: relative;
+    border-radius: 0.5rem;
+    width: 95%;
+    height: 66%;
+    top: 6.75%;
+    left: 2.75%;
+  }
+
+  .content :global(.is-brand-background) {
+    width: 100%;
+    height: 100%;
+    position: absolute;
+  }
+
+  .backgroundPosterButton {
+    position: absolute;
+    z-index: 1;
+    bottom: 0;
+    left: 0;
+  }
+
+  .deleteButtonWrapper {
+    display: flex;
+    justify-content: flex-end;
+    position: absolute;
+    bottom: 0;
+    height: 85%;
+    width: 100%;
+    z-index: 1;
+  }
+
+  .deleteButton {
+    top: 0;
+  }
+
+  .slider {
+    z-index: 2;
+    display: flex;
+    position: absolute;
+    bottom: 0;
+    border-radius: 0.5rem;
+  }
+
+  /*********** Baseline, reset styles ***********/
+  input[type='range'] {
+    -webkit-appearance: none;
+    appearance: none;
+    background: transparent;
+    cursor: pointer;
+    width: 10rem;
+  }
+
+  /* Removes default focus */
+  input[type='range']:focus {
+    outline: none;
+  }
+
+  /******** Chrome, Safari, Opera and Edge Chromium styles ********/
+  /* slider track */
+  input[type='range']::-webkit-slider-runnable-track {
+    background-color: #e6e6e6;
+    border-radius: 0.5rem;
+    height: 0.5rem;
+  }
+
+  /* slider thumb */
+  input[type='range']::-webkit-slider-thumb {
+    -webkit-appearance: none; /* Override default look */
+    appearance: none;
+    margin-top: -8px; /* Centers thumb on the track */
+    background-color: #828282;
+    border-radius: 0.5rem;
+    height: 1.5rem;
+    width: 0.5rem;
+  }
+
+  input[type='range']:focus::-webkit-slider-thumb {
+    outline: 3px solid #828282;
+    outline-offset: 0.125rem;
+  }
+
+  /*********** Firefox styles ***********/
+  /* slider track */
+  input[type='range']::-moz-range-track {
+    background-color: #e6e6e6;
+    border-radius: 0.5rem;
+    height: 0.5rem;
+  }
+
+  /* slider thumb */
+  input[type='range']::-moz-range-thumb {
+    background-color: #828282;
+    border: none; /*Removes extra border that FF applies*/
+    border-radius: 0.5rem;
+    height: 1.5rem;
+    width: 0.5rem;
+  }
+
+  input[type='range']:focus::-moz-range-thumb {
+    outline: 3px solid #828282;
+    outline-offset: 0.125rem;
+  }
```

Comment:
Lots of custom CSS for making sliders. Just use https://wikiki.github.io/form/slider/.